CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
-## [Unreleased]
+## [0.1.1] - 04-Nov-2022
```

Comment:
Totally a matter of preference, but I like to have my change logs for gems be a single line with a URL pointing to the releases page (https://github.com/gap777/automatic_namespaces/releases).

Reason being that a release is made for each version with a list of the PRs included in the release (since your CD uses auto generate release notes), so it looks like a change log anyways (and can be edited later)!